easybuild/tools/github.py

```diff
@@ -617,9 +619,14 @@ def _easyconfigs_pr_common(paths, start_branch=None, pr_branch=None, target_acco
 
     _log.debug("git status: %s", git_repo.git.status())
 
-    # copy files to right place
+    deps = resolve_dependencies([process_easyconfig(ec)[0] for ec in paths], modules_tool())
```

Comment:
@Caylo make sure a proper error is printed here in case dependencies could not be resolved with the current robot search path